hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/test/resources/missing-downstream-entry-spans/after-enrichment.json

```diff
@@ -79,6 +79,14 @@
               "binary_value": null,
               "value_list": null,
               "value_map": null
+            },
+            "enriched.internalDurationMillis":{
+              "value": {
+                "double": 4041
```

Comment:
should we keep the data type consistent? like above its string.

Reply:
Metrics shouldn't be in strings. I store it here for easier processing later.